parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestParquetWriter.java

```diff
@@ -108,7 +107,8 @@ public void test() throws Exception {
           assertEquals(1.0f, group.getFloat("float_field", 0), 0.001);
           assertEquals(2.0d, group.getDouble("double_field", 0), 0.001);
           assertEquals("foo", group.getBinary("flba_field", 0).toStringUsingUTF8());
-          assertEquals(Binary.fromByteArray(new byte[12]), group.getInt96("int96_field", 0));
+          assertEquals(Binary.fromConstantByteArray(new byte[12]), group.getInt96("int96_field",
+              0));
```

Comment:
Nit: it's better to insert new lines further up the syntax tree. For example, after the first argument to assertEquals rather than in the call to getInt96.